projects/wmf_default.yaml

```diff
@@ -63,6 +63,8 @@ paths:
                                 "ttl": 7200}, options.mobileapps)}}'
                 - path: v1/announcements.yaml
                   options: '{{merge({"announcement_cache_control": "s-maxage=86400, max-age=86400"}, options.mobileapps)}}'
+                - path: v1/trending.js
```

Comment:
Hm hm. I wonder if we should now split enwiki off, seeing that trending data is available only for enwiki (for now).